cmake/gcovr.cmake

```diff
@@ -0,0 +1,16 @@
+if(USE_GCOV)
```

Comment:
We should create a option in the main `CMakeLists.txt` for this variable. Ideally we should reuse the existing one for remote coverage `BUILD_WITH_COVERAGE`.

Reply:
Note that the line `include(cmake/gcovr.cmake REQUIRED)` should appear after that option.

Reply:
@piponazo, should we continue using the new `gcovr.cmake` file, or should we append the code to `BUILD_WITH_COVERAGE` within the `compilerFlags.cmake` file?

Reply:
I do not have a strong opinion about that. Either you add it in `compilerFlags.cmake` well separated from the "generic" compiler flags or you keep it in that new file. Up to you 😉